### ISSUE TITLE
Debian repo is using deprecated SHA1 signatures

### ISSUE THREAD

**Issue by ghost** [bazelbuild/bazel#1611]
On Debian testing, apt-get reports that the Bazel repo is using weak signature algorithms:

```
~ $ sudo apt-get update
Hit:1 http://storage.googleapis.com/bazel-apt stable InRelease
Reading package lists... Done
W: http://storage.googleapis.com/bazel-apt/dists/stable/InRelease: Signature by key 71A1D0EFCFEB6281FD0437C93D5919B448457EE0 uses weak digest algorithm (SHA1)
```

Debian docs: https://wiki.debian.org/Teams/Apt/Sha1Removal

**Comment by xpe** [comment]
Per https://wiki.debian.org/Teams/Apt/Sha1Removal

> This page keeps track of repositories broken or half-broken by the SHA1 removal. Please note that we intend to shut off SHA1 completely on January 1, 2017.

Who at Google can help?

**Comment by damienmg** [comment]
@meteorcloudy: will you have time to look at that?

**Comment by meteorcloudy** [comment]
OK, I will.

**Comment by bbarenblat** [comment]
I still show a SHA-1 signature on the Bazel repository:

```
# apt update
...
Hit:2 http://storage.googleapis.com/bazel-apt stable InRelease                                                  
...
W: http://storage.googleapis.com/bazel-apt/dists/stable/InRelease: Signature by key 71A1D0EFCFEB6281FD0437C93D5919B448457EE0 uses weak digest algorithm (SHA1)
```

**Comment by meteorcloudy** [comment]
I see, the reason is my fix didn't make into Bazel 0.4.1. It should be fixed in the next release. :)
